src/modules/lua_general.c

```diff
@@ -502,14 +502,35 @@ lua_hook_varargs(lua_State *L, const char *proto, va_list ap) {
     else if(mtev_hash_retrieve(&lua_ctypes, args[i], strlen(args[i]), &vfunc)) {
       mtev_lua_push_dynamic_ctype_t func = vfunc;
       func(L, ap);
-      //mtev_lua_setup_http_ctx(L, ctx);
     } else {
-      mtevL(nlerr, "unknown hook parameter type: '%s' consider mtev_lua_register_dynamic_ctype(...)\n", args[i]);
-      if(is_ptr) {
-        void *ptr = va_arg(ap, void *);
-        lua_pushlightuserdata(L,ptr);
+      bool is_ffi = false;
+      if(!is_ptr) {
+        /* we can't proceed b/c future arguments can't be popped off the varargs list correctly */
+        return i-1;
```

Comment:
Should log an error here before returning.

Reply:
hooks can fire fast! I made it a debug message so it's easier to silence.